### ISSUE TITLE
repository name confusing

### ISSUE THREAD

**Issue by kostrzewa** [HISKP-LQCD/sLapH-projection-NG#1]
is the idea to combine the contraction and projection?

**Comment by maowerner** [comment]
NG stands for NextGeneration apparently.

The idea is to analytically (Mathematica) perform the Isospin and Spin-projection and Wick contraction. The code is supposed to take representation matrices and continuum operators and returns a list of hdf5datasetnames along with coefficients they need to be added up. 

The output thus is an interface between the contraction code and the LuescherAnalysis, replacing my sLapH-projection code.

I am not sure whether @martin-ueding wants to actually perform the projection on the data in Mathematica, but I assume that will be in R.

**Comment by martin-ueding** [comment]
> is the idea to combine the contraction and projection?

No, I've just been writing “sLapH-contraction” too often and when I tried to write “sLapH-projection” I failed on the repository name. The one on my disk has the correct name at least 😅. I fixed that now.

As Markus wrote, my plan is to have the projection produce a data frame which tells us which HDF5 datasets to extract and combine with which factors. See Section 1 in [New_Subduction.pdf](https://github.com/HISKP-LQCD/sLapH-projection-NG/files/2816637/New_Subduction.pdf).

> NG stands for NextGeneration apparently.

Yes, I've seen that in a bunch of projects. Likely this is just the work-in-progress name until we come up with some truly awe-inspiring or get bought by a corporation and call it something like “Acme Corp Widget Frobnicator”.

> I am not sure whether @martin-ueding wants to actually perform the projection on the data in Mathematica, but I assume that will be in R.

See Section 3.4, where I have listed C++, Python and R as possible candidates for that step. I expect this part of the code to be perhaps 300 lines of code, and I think either language is fine. So we might as well use R as now all people in our group have experience with it.
